.github/action_scripts/delegated_staking/delegated-staking.js

```diff
@@ -414,152 +320,50 @@ const checkCreateNodeParameters = async (urls) => {
   )
   logWorkflow.info('Update second node params is OK')
 
-  const bothNodesParams = await getNodeParams(urls)
```

Comment:
This file is getting extremely large, it makes the most sense to split the node params request into a different file since they're somewhat independent.

Reply:
Cool - I like that. It makes it more explicit that node params are being tested also.